web/run.py

```diff
@@ -91,6 +91,13 @@ def check_if_token_is_revoked(decrypted_token):
     login_manager.login_message = "You must be logged in to access this page!!!"
     login_manager.login_view = "auth.login"
 
+    # CORS
+    @app.after_request
```

Comment:
This currently adds the header on every page of the web GUI, doesn't it? I think it should be limited to the `/api/` paths alone, as this could be used for cross-site request forgery (CSRF) attacks; e.g. the admin pages could be used by another site while an administrator accessing the malicious site is logged in.

Reply:
Just noticed that the API has administrative features ([web/restapi/admin.py](https://github.com/cve-search/cve-search/blob/master/web/restapi/admin.py)), too, so this should be further limited to the read only API paths.

Reply:
Should be fixed now.